### ISSUE TITLE
Where is my LIZARD!?

### ISSUE THREAD

**Issue by ChristianIvarsson** [notepad-plus-plus/notepad-plus-plus#10715]
### Description of the Issue
Somewhere inbetween the last release of np 7 and the current version there's been a horrible regression where the legendary lizard has been removed from the selectable icons so it's no longer possible to change back to the old icon.

I don't care if it's "ugly", it had soul and is very dear to many of us that have been using np++ for many years.

#pleasebringlizardback

**Comment by hc970** [comment]
Hi Christianlvarsson,

Thanks for raising this issue.

I noticed the same issue with the icon occurred from version 8.1.3 onwards.

Very similar to your workaround, I downloaded the portable version of 8.1.2 and extracted it to another folder and deleted everything but notepad++.exe.

I then created a shortcut to the latest version of notepad++.exe (I now have 8.1.9 installed) and changed the properties of the shortcut to point to the 8.1.2 version of notepad++.exe for the icon  I have only needed to do this once since as newer versions are installed they will still be called notepad++.exe and  thus the shortcut remains valid.

I totally agree about the lizard having a soul. He/she has been there since I started using Notepad++ back in 2006 and I agree the lizard is dear to me too :)

Thanks.
